app/views/invoices/index.erb

```diff
@@ -6,7 +6,7 @@
           <a href="/invoices/<%= invoice.id %>">Invoice: <%= invoice.id %></a>
 
         <form action='/invoices/<%= invoice.id %>/edit' method='get'>
-          <input type='submit' value="Edit" name='Edit'>
+          <!-- <input type='submit' value="Edit" name='Edit'> -->
```

Comment:
do we need the commented out part?

Reply:
Ugh! Forgot to delete the commented out line after I ran the test.  I was trying to make sure I included the disabled keyword.  It can be totally removed and it will still work :)